src/content/upgrades/beacon-chain/index.md

```diff
@@ -7,51 +7,43 @@ sidebar: true
 image: ../../../assets/upgrades/core.png
 summaryPoint1: The Beacon Chain doesn't change anything about the Ethereum we use today.
 summaryPoint2: It introduced proof-of-stake to the Ethereum ecosystem.
-summaryPoint3: It will coordinate the network, serving as the consensus layer.
-summaryPoint4: It is an essential precursor to upcoming scaling upgrades, such as sharding.
+summaryPoint3: It was merged with the original Ethereum proof-of-work chain in 2022.
+summaryPoint4: The consensus logic and block gossip protocol introduced by the Beacon Chain now secures Ethereum.
```

Comment:
```suggestion
summaryPoint4: The Beacon Chain introduced consensus logic and block gossip protocol which now secures Ethereum.
```
Not sure if this is great but I feel like re-wording this to more of an active voice would be better.